exporters/trace/README.md

```diff
@@ -27,7 +29,7 @@ config:
 <dependency>
   <groupId>com.google.cloud.opentelemetry</groupId>
   <artifactId>exporter-trace</artifactId>
-  <version>0.11.0</version>
+  <version>0.14.0</version>
```

Comment:
Should we put a placeholder so we don't need to bump each time?

Reply:
It's a good suggestion.

TL;DR: not worth the hassle (for users or us).

Longer answer:

- We should give users copy-pastable deps, it's a nice thing.
- I couldn't find any way to auto-update this WITHOUT issuing a commit
- We can probably add this to our automation but it's not the end of the world right now.